publish/dir.props

```diff
@@ -14,13 +19,27 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <InstallerFile Include="$(PackagesOutDir)*$(SharedFrameworkNuGetVersion)$(InstallerExtension)" />
-    <InstallerFile Include="$(PackagesOutDir)*$(SharedFrameworkNuGetVersion)$(CombinedInstallerExtension)" />
-    <CompressedFile Include="$(PackagesOutDir)*$(SharedFrameworkNuGetVersion)$(CompressedFileExtension)" />
-    <RuntimePackageFile Include="$(PackagesOutDir)runtime.*$(SharedFrameworkNuGetVersion).nupkg" />
-    <RuntimePackageFile Include="$(PackagesOutDir)runtime.*$(SharedFrameworkNuGetVersion).symbols.nupkg" />
-    <RidAgnosticPackageFile Include="$(PackagesOutDir)*$(SharedFrameworkNuGetVersion).nupkg" Exclude="@(RuntimePackageFile)" />
-    <RidAgnosticPackageFile Include="$(PackagesOutDir)*$(SharedFrameworkNuGetVersion).symbols.nupkg" Exclude="@(RuntimePackageFile)" />
+    <CompressedFile Include="$(PackagesOutDir)**/*$(CompressedFileExtension)">
+      <RelativeBlobPath>$(BinariesRelativePath)</RelativeBlobPath>
+    </CompressedFile>
+    <RuntimePackageFile Include="$(PackagesOutDir)**/runtime.*.nupkg" >
+      <RelativeBlobPath>$(BinariesRelativePath)</RelativeBlobPath>
+    </RuntimePackageFile>
+    <RuntimePackageFile Include="$(PackagesOutDir)**/runtime.*.symbols.nupkg" >
+      <RelativeBlobPath>$(BinariesRelativePath)</RelativeBlobPath>
+    </RuntimePackageFile>
+    <RidAgnosticPackageFile Condition="'$(PublishRidAgnosticPackages)' == 'true'" Include="$(PackagesOutDir)**/*.nupkg" Exclude="@(RuntimePackageFile)" >
```

Comment:
This being conditional on PublishRidAgnosticPackages being true seems weird.  Instead, I would always add stuff to the RidAgnosticsPackageFile ItemGroup but only consume it when PublishRidAgnosticPackages is set to true.